libs/wire-subsystems/src/Wire/FederationConfigStore.hs

```diff
@@ -24,8 +24,8 @@ data AddFederationRemoteTeamResult
   | AddFederationRemoteTeamDomainNotFound
   | AddFederationRemoteTeamRestrictionAllowAll
 
--- TODO: This store effect is more than just a store, we should break it up in
--- business logic and store
+-- FUTUREWORK: This store effect is more than just a store,
+-- we should break it up in business logic and store
```

Comment:
Is this still work for this ticket? We moved the code from brig to wire-subsystems in this PR so it might be confusing for someone to see that it wasn't made to fit the nomenclature here.

Reply:
> Is this still work for this ticket? We moved the code from brig to wire-subsystems in this PR so it might be confusing for someone to see that it wasn't made to fit the nomenclature here.

Is this a blocker for you or is a follow-up ticket OK?

Reply:
Follow up is also ok :+1: